nautobot/dcim/choices.py

```diff
@@ -1578,3 +1578,14 @@ class VirtualDeviceContextStatusChoices(ChoiceSet):
         (STATUS_ACTIVE, "Active"),
         (STATUS_PLANNED, "Planned"),
     )
+
+
+#
+# Controllers
+#
+
+
+class ControllerCapabilitiesChoices(ChoiceSet):
+    WIRELESS = "wireless"
+
+    CHOICES = ((WIRELESS, "Wireless"),)
```

Comment:
Do we have a clear picture at this time what other capabilities might logically be added to this choiceset?

Reply:
We do not at this time, though John would like this to be a Choice field for future implementations. But this is a good field to discuss as Cristian would like for an App to be able to extend the choices.